### PR TITLE
Update foreman-compute fog-aws and fog dependencies

### PR DESCRIPTION
**Pull request by eLobato**

### CODE REVIEW

File: foreman/foreman.spec [hunk 1]
```diff
@@ -258,7 +258,7 @@ Meta Package to install requirements for ovirt support
 %package compute
 Summary: Foreman Compute Resource support via fog
 Group:  Applications/System
-Requires: %{?scl_prefix}rubygem(fog) = 1.27.0
+Requires: %{?scl_prefix}rubygem(fog) = 1.28.0
 Requires: %{?scl_prefix}rubygem(fog-core) = 1.27.4
```
Comment (domcleal) [inline foreman/foreman.spec]: 1.29.0 to match https://github.com/theforeman/foreman/pull/2183/files

File: foreman/foreman.spec [hunk 2]
```diff
@@ -276,8 +276,8 @@ particular, OpenStack and Rackspace.
 %package ec2
 Summary:   Foreman Amazon Web Services (AWS) EC2 support
 Group:     Applications/System
-Requires:  %{?scl_prefix}rubygem(fog-aws) >= 0.0.8
-Requires:  %{?scl_prefix}rubygem(fog-aws) < 0.1.0
+Requires:  %{?scl_prefix}rubygem(fog-aws) >= 0.1.0
+Requires:  %{?scl_prefix}rubygem(fog-aws) < 0.2.0
```
Comment (domcleal) [inline foreman/foreman.spec]: `< 1.0.0` to match https://github.com/theforeman/foreman/pull/2183/files

**Comment by domcleal** [comment]
Ignore the test failure on this one.. the test job in Jenkins can't handle the core foreman package(s) yet.

**Comment by eLobato** [comment]
@domcleal Done, thanks for the heads up

File: foreman/foreman.spec [hunk 3]
```diff
@@ -258,8 +258,8 @@ Meta Package to install requirements for ovirt support
 %package compute
 Summary: Foreman Compute Resource support via fog
 Group:  Applications/System
-Requires: %{?scl_prefix}rubygem(fog) = 1.27.0
-Requires: %{?scl_prefix}rubygem(fog-core) = 1.27.4
+Requires: %{?scl_prefix}rubygem(fog) = 1.28.0
+Requires: %{?scl_prefix}rubygem(fog-core) = 1.29
```
Comment (domcleal) [inline foreman/foreman.spec]: 1.29.0

**Comment by domcleal** [comment]
Merged as 19831be, thanks @eLobato!

**Pull request closed without merging**